autopush/db.py

```diff
@@ -52,6 +52,12 @@ def hasher(uaid):
     return uaid
 
 
+def normalize_id(id):
+    if not id:
+        return id
+    return str(uuid.UUID(id))
```

Comment:
This is an expensive operation. Might want to have a regex for the normalized ID, so that re-normalizing a normalized one isn't done, which would be faster if its generally the case that they're already normalized. If it's rare that they're normalized, the extra regex would be slower... so whichever.